### ISSUE TITLE
Двусторонняя телеметрия мавлинк?

### ISSUE THREAD

**Issue by whoim2** [baychi/OpenExpertTX#1]
Как я понимаю, она отсутствует в этой лрс, верно?

**Comment by baychi** [comment]
Верно, отсутствует. С уважением, Александр -------- Исходное сообщение --------От: whoim2 ***@***.***>Дата: пн, 12 июл. 2021 г., 15:27Кому: baychi/OpenExpertTX ***@***.***>Копия: Subscribed ***@***.***>Тема: [baychi/OpenExpertTX] Двусторонняя телеметрия мавлинк? (#1)

Как я понимаю, она отсутствует в этой лрс, верно?

**Comment by whoim2** [comment]
Тогда меня спасет ветка openlrsng/gitsly?
